### ISSUE TITLE
Topics for discussion

### ISSUE THREAD

**Issue by jorainer** [lgatto/Spectra#27]
Some topics to discuss regarding `Spectra` at the devcall (https://github.com/lgatto/MSnbase/issues/471)

- Mention: for `Spectra` key to allow processing of things in parallel, separate for each file.
- Mention: nice to have the possibility to add arbitrary spectra variables.
- Question: implementation of `MsBackendRleDataFrame`: virtual class? unexported class?
- Question: *peaks* functions (e.g. https://github.com/lgatto/Spectra/blob/jomaster/R/Spectra-functions.R#L91-L112) successor of `Spectrum` functions in `MSnbase`. Any concerns?
- TODO: implement a `peaksapply,Spectra` method to apply any `FUN` to the peak data.
- TODO: applying processing steps to peaks data: `applyProcessingQueue,Spectra`?
